src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -1398,7 +1394,27 @@ bool SlaterDetBuilder::readDetListH5(xmlNodePtr cur,
   CIcoeff.resize(ndets);
   ConfigTag.resize(ndets);
 
+#if defined(QMC_COMPLEX)
+  std::vector<double> CIcoeff_real;
+  std::vector<double> CIcoeff_imag;
+  std::complex <double> im(0,1);
+  CIcoeff_imag.resize(ndets);
+  CIcoeff_real.resize(ndets);
+
+  hin.read(CIcoeff_real, "Coeff");
+  hin.read(CIcoeff_imag, "Coeff_imag");
+
+
+  for (size_t i=0; i<ndets;i++)
+  {
+      CIcoeff[i]=CIcoeff_real[i]+im*CIcoeff_imag[i];
```

Comment:
Can we do `CIcoeff[i] = ValueType(CIcoeff_real[i], CIcoeff_imag[i])`. Also just remove the `{}` which is not needed for 1 line.

Reply:
Done